src/dsp_tools/utils/project_create.py

```diff
@@ -35,51 +39,51 @@ def _create_project_on_server(
         verbose: verbose switch
 
     Raises:
-        BaseError: if the project cannot be created on the DSP server
+        UserError: if the project cannot be created on the DSP server
 
     Returns:
         a tuple of the remote project and the success status (True if everything went smoothly, False otherwise)
     """
     try:
         # the normal, expected case is that this try block fails
         project_local = Project(con=con, shortcode=project_definition["project"]["shortcode"])
-        project_remote: Project = try_network_action(
-            action=lambda: project_local.read(),
-            failure_msg=""
-        )
-        print(f"\tWARNING: Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP "
-              f"server. Updating it...")
-        success = False
+        project_remote: Project = try_network_action(lambda: project_local.read())
+        print(f"\tWARNING: Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP server. Updating it...")
+        logger.warning(f"Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP server. Updating it...")
```

Comment:
```suggestion
        msg = f"Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP server. Updating it..."
        print(f"\tWARNING: {msg}")
        logger.warning(msg)
```

just a thought, how to make this duplication a bit nicer